.github/ISSUE_TEMPLATE/server-migration.md

```diff
@@ -60,44 +63,46 @@ Make sure to clear cache so that instance settings are applied:
 
 ## 3. Migration
 ### preparation
-- [ ] **new server**: `bin/rake db:reset -e production` (important: make sure you're on the new server!)
-- [ ] `deploy.yml -l x_prod2 -e "git_version=vX.Y.Z"` matching version with current prod
+- [ ] Reset database on new server, to avoid any migration issues due to being out of sync
+  `bin/rake db:reset` (You will need to confirm. Make sure you're on the new server!)
+- [ ] Update ansible_host IP in `inventory/hosts` and ensure provision works (this will update host in `.env.production`).
+    `ansible-playbook playbooks/provision.yml -l x_prod`
+- [ ] `ansible-playbook playbooks/deploy.yml -l x_prod -e "git_version=vX.Y.Z"` matching version with current prod
 - [ ] old server: make a tiny data change to verify later (eg add `.` in meta description `/admin/general_settings/edit`)
 
 ### switchover: old server
-- [ ] 🚧 `maintenance_mode.yml`
+- [ ] 🚧 `ansible-playbook playbooks/maintenance_mode.yml -l x_prod`
 - [ ] `sudo systemctl stop sidekiq redis-jobs puma`
+- [ ] `ansible-playbook -l x_prod -e rsync_to=x_prod2 playbooks/db_transfer.yml &&`
+- [ ] `ansible-playbook -l x_prod -e rsync_to=x_prod2 playbooks/transfer_assets.yml`
 - [ ] Transfer `/var/lib/redis-jobs/dump.rdb` to new server (see guide)
-- [ ] `db_transfer.yml` ~3min
-- [ ] `sudo systemctl stop postgres` (ensure other integrations no longer touch it)
-- [ ] `transfer_assets.yml` just in case
+- [ ] `sudo systemctl stop postgresql` (ensure other integrations no longer touch it)
 
 ### switchover: new server
 - [ ] `sudo systemctl restart puma; sudo systemctl start sidekiq redis-jobs`
```

Comment:
Sidekiq is disabled at this point. We have to enable it first before we can start it. But I personally would probably do it staggered:

* Start puma and check with hosts file.
* Start sidekiq and check log file.
* Then install proxy forwarding.

Reply:
How much time did you need for testing? If it's short, maybe we could do it during maintenance mode which would enable us to then switch over straight away. It would save us all the work of transferring data twice and resetting in between.